docs/install/upgrade-installation.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Upgrading your installation"
+weight: 21
+type: "docs"
+---
+
+If you already have Knative installed, upgrade your installation by running the
+install command for the new version. We recommend upgrading by a single version
+number. For example, if you have v0.6.0 installed, you should upgrade to v0.7.0
+before attempting to upgrade to v0.8.0. To verify the version number you
+currently have installed, see
+[Checking your installation version](./check-install-version.md).
+
+## Breaking changes
+
+Breaking changes between Knative versions are documented in the Knative release
+notes. Before upgrading, review the release notes for the target version to
+learn about any changes you might need to make to your Knative applications.
+
+The [Serving](https://github.com/knative/serving/releases) and
+[Eventing](https://github.com/knative/eventing/releases) release notes are
+published with each version on the "Releases" page of their respective
+repositories in GitHub.
+
+## How to upgrade
+
+To upgrade, apply the `.yaml` files for the new Knative version of your desired
+Knative components and features to your Kubernetes cluster. For example, for a
+cluster running v0.10.0 of the Knative Serving component, the  following command
+upgrades the installation to v0.11.0:
```

Comment:
This is the way to go if the customer originally installed Knative via the direct YAML. If they installed Knative through the Operator package the instructions will be different. We should call that out.

Reply:
I added a note on that to beginning. I need to look and see if there's a place we can point to on upgrading with the operator.